indico/modules/events/registration/__init__.py

```diff
@@ -77,9 +78,22 @@ def _inject_event_header(event, **kwargs):
                 .order_by(db.func.lower(RegistrationForm.title))
                 .all())
 
-    participants = get_unique_published_registrations(event)
+    # A participant could appear more than once in the list in case he register to multiple registration form.
+    # This is deemed very unlikely in the case of meetings and lectures and thus not worth the extra complexity.
+    registrations = (Registration
+                     .find(Registration.is_active,
+                           ~RegistrationForm.is_deleted,
+                           RegistrationForm.event_id == event.id,
+                           RegistrationForm.publish_registrations_enabled,
+                           _join=Registration.registration_form)
+                     .order_by(db.func.lower(Registration.first_name),
+                               db.func.lower(Registration.last_name),
+                               Registration.id)
+                     .options(contains_eager('registration_form')) #FIXME Sort here
```

Comment:
instead of the options call you could also use `_eager=Registration.registration_form` in the `find()` call; didn't think about this before.

https://github.com/indico/indico/blob/master/indico/core/db/sqlalchemy/util/models.py#L35